plugins/orchestrator-backend/src/types/pagination.ts

```diff
@@ -0,0 +1,17 @@
+import { Request } from 'express-serve-static-core';
+
+export interface Pagination {
+  offset?: number;
+  limit?: number;
+  order?: string;
+  sortField?: string;
+}
+
+export function buildPagination(req: Request): Pagination {
+  return {
+    offset: req.query.pageNumber ? Number(req.query.pageNumber) : 0,
+    limit: req.query.pageSize ? Number(req.query.pageSize) : 10,
+    sortField: req.query.sortField ? String(req.query.sortField) : undefined,
+    order: req.query.order ? String(req.query.order) : 'ASC',
```

Comment:
Could you please extract the default values into constants? `0`, `10`, `undefined` and `ASC`.

Reply:
[b32bee1](https://github.com/janus-idp/backstage-plugins/pull/1164/commits/b32bee198b65ada69cf4f4182e1fc82aaf7823e0)